startGame.js

```diff
@@ -0,0 +1,29 @@
+document.getElementById("button").addEventListener('click', function(){
```

Comment:
Preferably use jQuery selectors please

Reply:
no more jquery in this file so will leave it